charts/kafka-operator/templates/crds.yaml

```diff
@@ -31,7 +33,6 @@ spec:
     listKind: KafkaClusterList
     plural: kafkaclusters
     singular: kafkacluster
-  preserveUnknownFields: false
```

Comment:
Let's set this explicitly to true cause the default may change.
And also it is more explicit this way

Reply:
Right, this is the way the `controller-gen` interprets the filed if set to `true` in the Makefile.
But I agree it is safer to set it explicitly.